code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -10,9 +10,17 @@
 
 	health = maxHealth - getBrainLoss()
 
-	//TODO: fix husking
-	if(((maxHealth - getFireLoss()) < config.health_threshold_dead) && stat == DEAD)
-		ChangeToHusk()
+	if(stat == DEAD)
+		if(maxHealth - getFireLoss() < config.health_threshold_dead)
```

Comment:
this won't work, because config.health_threshold_dead is defunct and not used for humans, only for simple mobs

also this entire chain makes no sense to me? `if(200 - fireloss < 200)`?